src/AppKit/NSFont.cs

```diff
@@ -173,10 +183,13 @@ public static NSFont ControlContentFontOfSize (nfloat fontSize)
 			}
 		}
  
-#if !NET
- 		[Deprecated (PlatformName.MacOSX, 10, 13)]
+#if NET
+ 		[UnsupportedOSPlatform ("macos10.13")]
+#if MONOMAC
+ 		[Obsolete ("Starting with macos10.13.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+#endif
 #else
-		[UnsupportedOSPlatform ("macos10.13")]
+ 		[Deprecated (PlatformName.MacOSX, 10, 13)]
```

Comment:
```suggestion
		[Deprecated (PlatformName.MacOSX, 10, 13)]
```
nit, extra space

Reply:
@chamons is your translator getting confused? Why is it adding spaces?

Reply:
This is because there is an unnecessary space before the `#if !NET` above that is getting processed as the leading space. I'll have to fix each line.